lint-staged.config.js

```diff
@@ -2,7 +2,7 @@
 module.exports = {
   '**/*.ts': () => 'yarn test:types',
   '!(website/)**/*.(js|mjs|ts|)': (filenames) =>
-    `yarn lint --fix ${filenames.join(' ')}`,
+    `eslint --ext .js,.ts --fix ${filenames.join(' ')}`,
```

Comment:
This is necessary because otherwise `turbo` runs `eslint` in each project not from the root.